testing/adios2/engine/dataman/CMakeLists.txt

```diff
@@ -17,6 +17,9 @@ foreach(tst IN ITEMS
   )
 endforeach()
 
+python_add_test(NAME Test.Engine.DataMan1D.Serial SCRIPT TestDataMan1D.py)
+python_add_test(NAME Test.Engine.DataMan1xN.Serial SCRIPT TestDataMan1xN.py)
+
```

Comment:
I think you just need to put these lines in a `if(ADIOS2_HAVE_Python)` condition. If you look at the cdash output, you'll see that python isn't turned on in the nvhpc build: https://open.cdash.org/build/8527861